tests/functions_tests/test_point_to_point_communication.py

```diff
@@ -100,3 +102,31 @@ def test_communication(self):
             err = chainermn.functions.send(
                 y, self.communicator, self.rank_send)
             err.backward()
+
+    def test_retain(self):
+        if self.communicator.rank == 0:
```

Comment:
Does this test work if more than 2 processes invoked?
It should be skipped if communicator.size > 2 ?